RELEASE/scripts/autoscend.ash

```diff
@@ -2300,57 +2300,62 @@ boolean questOverride()
 
 boolean L13_towerNSNagamar()
 {
+	//status != 12 is an override in case auto_wandOfNagamar is incorrectly set to false
+	//status 11 means ready to fight sorceress, status 12 means you lost to her because you lacked a wand of nagamar
```

Comment:
Took me a while to figure out what's going on here: Perhaps a comment like "If we've already lost to the sorceress due to lacking a wand, skip and go to the garves instead." ? Assuming that's an accurate summary of what's happening here.

Reply:
clarified the comments in commit 91bb56c